test/e2e/apimachinery/aggregator.go

```diff
@@ -161,12 +166,21 @@ func TestSampleAPIServer(f *framework.Framework, image, namespaceName string) {
 	ns := f.Namespace.Name
 	if framework.ProviderIs("gke") {
 		// kubectl create clusterrolebinding user-cluster-admin-binding --clusterrole=cluster-admin --user=user@domain.com
-		framework.BindClusterRole(client.RbacV1beta1(), "cluster-admin", ns,
-			rbacv1beta1.Subject{Kind: rbacv1beta1.ServiceAccountKind, Namespace: ns, Name: "default"})
-		err := framework.WaitForAuthorizationUpdate(client.AuthorizationV1beta1(),
-			serviceaccount.MakeUsername(ns, "default"),
-			"", "get", schema.GroupResource{Group: "storage.k8s.io", Resource: "storageclasses"}, true)
-		framework.ExpectNoError(err, "Failed to update authorization: %v", err)
+		//subject := *f.ClientSubject
+		authenticated := rbacv1beta1.Subject{Kind: rbacv1beta1.GroupKind, Name: user.AllAuthenticated}
+		framework.Logf("WRF - setting permissions on %v.", authenticated)
+		framework.BindClusterRole(client.RbacV1beta1(), "cluster-admin", ns, authenticated)
+		// TODO: cheftako - Get this working with whoami/ClientSubject & WaitForAuthorizationUpdate
+		/*var name string
+		if subject.Kind == rbacv1beta1.ServiceAccountKind {
+			name = serviceaccount.MakeUsername(ns, subject.Name)
+		} else {
+			name = subject.Name
+		}
+		err := framework.WaitForAuthorizationUpdate(client.AuthorizationV1beta1(), name, "",
+			"create", schema.GroupResource{Group: "rbac.authorization.k8s.io", Resource: "clusterroles"}, true)
+		framework.ExpectNoError(err, "Failed to update authorization for '%v': %v", f.ClientSubject, err)*/
+		time.Sleep(10 * time.Second)
```

Comment:
Its a guess. One of the reasons the TODO is there. We *should* be waiting to see the authorization update (the commented out code). However that was not seeing the change. Once we get 1.8 unblocked I will go back and work out why its not working.